src/strong17.lib/strong17/Strong.inspect.h

```diff
@@ -17,10 +17,10 @@ constexpr auto hasTag(Type<Strong<Value, Tags...>>, Type<CandidateTag> = {}) {
     return (std::is_same_v<CandidateTag, Tags> || ...);
 }
 
-/// tag inspection
-template<class CandidateTag, class Value, class... Tags>
-constexpr auto contains(Type<Strong<Value, Tags...>>, Type<CandidateTag> = {}) {
-    return (std::is_same_v<CandidateTag, Tags> || ...);
+/// type inspection
+template<class CandidateType, class Value, class... Tags>
+constexpr auto hasType(Type<Strong<Value, Tags...>>, Type<CandidateType> = {}) {
+    return std::is_same_v<Value, CandidateType>;

```

Comment:
ist a `hasType` function really necessary? - we have `extractValueType` already and it's easy to compare two types.